internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -755,6 +772,16 @@ func (r *BMACReconciler) reconcileSpokeBMH(ctx context.Context, log logrus.Field
 		Namespace: agent.Spec.ClusterDeploymentName.Namespace,
 		Name:      fmt.Sprintf(adminKubeConfigStringTemplate, cd.Name),
 	}
+	isNonePlatform, err := r.isNonePlatform(ctx, cd)
+	if err != nil {
+		log.WithError(err).Error("Is none platform")
```

Comment:
Please, could you use a more descriptive error?

Reply:
OK